src/domain/entities/User.ts

```diff
@@ -1,18 +1,40 @@
 import _ from "lodash";
+import { FutureData } from "./Future";
 import { NamedRef } from "./Ref";
 
-export interface User {
+export interface OrgUnitAccess {
+    orgUnitId: string;
```

Comment:
[minor] typically this id fields use the type `Id` from Ref.ts (which ofc it's a string). Example of Base.ts of other projects:

https://cs.github.com/EyeSeeTea/d2-reports/blob/9d32123ca07aedf7346a43d6d621d4f1320fa5ff/src/domain/common/entities/Base.ts?q=base.ts+namedref

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/pull/35/commits/c664c49dc8abcbeb0baeceac61d29b7d7275cb08